flow/util/genMetrics.py

```diff
@@ -467,19 +418,19 @@ def extract_metrics(cwd, platform, design, flow_variant, output):
                        "Total +\S+ +(\S+) +\S+ +\S+ +\S+",
                        logPath+"/6_report.log")
 
-    extractTagFromFile("finish__power__leak__total", metrics_dict,
+    extractTagFromFile("finish__power__leakage__total", metrics_dict,
                        "Total +\S+ +\S+ +(\S+) +\S+ +\S+",
                        logPath+"/6_report.log")
 
     extractTagFromFile("finish__power__total", metrics_dict,
                        "Total +\S+ +\S+ +\S+ +(\S+) +\S+",
                        logPath+"/6_report.log")
 
-    extractTagFromFile("finish__area", metrics_dict,
+    extractTagFromFile("finish__design__instance__area", metrics_dict,
                       "^Design area (\S+) u\^2",
                        logPath+"/6_report.log")
 
-    extractTagFromFile("finish__util", metrics_dict,
+    extractTagFromFile("finish__design__instance__utilization", metrics_dict,
```

Comment:
Why are there a bunch of keys with `design` in the name? Does that really add info? e.g. `finish__design__instance__utilization` vs. `finish__instance__utilization`

Reply:
In Metrics 2.1 format, we have a notion of "stage" and metrics category.  In this case "finish" is the stage  and "design" is the metrics category that hold all  physical information.  The current categories  are "design", "clock", "timing", "route" and "power".  The default stages are   run | init | synth | floorplan | globalplace | placeopt | detailedplace | cts | globalroute | detailedroute | final.   Stages can also be user defined like "user specific" snapshot

Reply:
I see, thanks.